packages/react-native/example/android/build.gradle

```diff
@@ -4,9 +4,9 @@ buildscript {
     ext {
         buildToolsVersion = "33.0.0"
         minSdkVersion = 24
-        compileSdkVersion = 34
-        targetSdkVersion = 34
-        kotlin_version = "1.8.0"
+        compileSdkVersion = 33
+        targetSdkVersion = 33
```

Comment:
Why were these versions got downgraded? Would it be better to keep them at 34 because of:
> Starting August 31 2024:
> 
> New apps and app updates must target Android 14 (API level 34) or higher to be submitted to Google Play; except for Wear OS and Android TV apps, which must target Android 13 (API level 33) or higher.

and their new rule of:
![image](https://github.com/user-attachments/assets/5867a67d-9440-4aa9-94e2-6f0a2cc5cc24)

Reply:
The example app was throwing errors specific to API 34. Apparently the version of React Native we use in the example doesn't support this. We should update it in another PR